### PR TITLE
Disable drift detection in performance

### PR DESCRIPTION
**Pull request by steven-sheehy**
**Description**:

* Bump performance to v0.113.1
* Disable drift detection in performance so reset works properly
* Remove grpc resource overrides that are now in 0.113
* Reduce initial disk size and pv size for citus

**Related issue(s)**:

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Comment by xin-hedera** [comment]
We may want to make the following changes now

- remove the additional 2 citus-coordinator nodes, and the additional3 citus-worker nodes
- rebuild the citus nodes with smaller initial disks, especially worker node disk, 512GB should be enough v.s. 3200GB as the default

File: clusters/preprod/common/helmrelease.yaml [hunk 1]
```diff
@@ -92,3 +92,7 @@ spec:
           cloud.google.com/neg: '{ "exposed_ports":{ "443":{"name": "mirrornode-preprod-new"} } }'
     zfs:
       enabled: true
+      coordinator:
+        initialDiskSize: 100GB
+      worker:
+        initialDiskSize: 1024GB
```
Comment (xin-hedera) [inline clusters/preprod/common/helmrelease.yaml:98]: this is more than what we need size wise.

however we have to bloat it up to get the IOPS we need for 10K tps tests in performance network

for balanced pd, the equation is

`iops = base + iops per GB * size in GB`

base is 3000, iops per GB for balanced PD is 6. We need somewhere around 9000 IOPS, so ended up to be 1024GB.

**Review by xin-hedera — APPROVED** [review]
LGTM

**Review by jnels124 — APPROVED** [review]
LGTM

**Pull request merged**